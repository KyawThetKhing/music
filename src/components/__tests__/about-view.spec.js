import About from '@/views/AboutView.vue'
import { mount } from '@vue/test-utils'

describe('AboutView.vue', () => {
  test('renders inner context', () => {
    const wrapper = mount(About, { shallow: true })
    expect(wrapper.text()).toContain('About View')
  })
})
