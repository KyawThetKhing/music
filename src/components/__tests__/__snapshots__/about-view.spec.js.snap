// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AboutView.vue > renders inner context 1`] = `
"<div>About View</div>
<base-button></base-button>"
`;
